qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -383,6 +473,10 @@ def save_file(self):
         # todo: how should this interact with project saving and workspaces when they are implemented?
         self.editor.save_current_file()
 
+    def open_manage_directories(self):
+        manage_directory_widget = ManageUserDirectories(self)
```

Comment:
I'm not sure we should keep the behaviour of having multiple dialogs open at once; I don't think it makes sense. Let's use `._exec_()` and block until we are finished.